src/localization/it.js

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// License: GNU GPLv3 or later. See the license file in the project root for more information.
+// Copyright © 2021 - present Aleksey Hoffman. All rights reserved.
+
+const description = {
+  name: 'Italiano',
+  locale: 'it'
+}
+
+const data = {
+  app_name: 'Sigma file manager',
+  locale_detected_message: 'Cambiare la lingua dell'app in inglese?',
+  page_home_title: 'Home Page',
+  page_navigator_title: 'Navigatore',
+  page_dashboard_title: 'Dashboard',
+  page_notes_title: 'Note',
+  page_settings_title: 'impostazioni',
+  settings_language_title: 'Lingua',
+  settings_language_options_title: 'Lingua dell'app',
+  settings_ui_zoom_title: 'Ridimensionamento della finestra',
+  settings_ui_zoom_options_title: 'livello di zoom UI | Corrente: ',
+  settings_ui_zoom_button_reset: 'Ripristina',
+  tooltip_button_shortcut: 'Scorciatoia',
+  tooltip_button_ui_zoom_decrease_title: 'Diminuisci il livello di zoom',
+  tooltip_button_ui_zoom_increase_title: 'Aumenta il livello di zoom',
+  tooltip_button_ui_zoom_reset_title: 'Ripristina il livello di zoom',
+  tooltip_button_full_screen_title: 'Schermo intero',
+  tooltip_text_to_copy: 'Copiare',
+  text_drive: 'Disco',
+  text_drives: 'Dischi',
+  text_global_search_title: 'Ricerca globale',
+  text_system_directories: 'Directory di sistema',
+  text_directory: 'directory',
+  text_file: 'file',
+  text_file_symlink: 'file-symlink',
+  text_directory_symlink: 'directory-symlink',
+  text_item: 'elemento',
+  text_item_plural: 'elementi',
+  text_size: 'Dimensione',
+  text_file_size: 'File size',
+  text_directory_size: 'Dimensione della directory',
+  text_get_size: 'Ottieni dimensioni',
+  text_path: 'Percorso',
+  text_real_path: 'Collega percorso di destinazione',
+  text_created: 'Creato',
+  text_modified: 'Contenuti modificati',
+  text_changed: 'Meta modificato',
+  text_protected: 'Protetto',
+  text_protected_win32: 'Protetto (all'interno di questa app)',
+  text_pinned: 'Appuntato',
+  text_mode: 'Permessi',
+  text_yes: 'Si',
+  text_no: 'No',
+  text_read_only: 'Solo lettura',
+  text_undo: 'annulla',
+  text_cancel: 'cancella',
+  text_clear_list_message: 'l'elenco degli elementi è stato cancellato',
+  text_workspace_added: 'l'area di lavoro è stata aggiunta',
+  text_allow_global_search_message: 'La ricerca globale ti consente di trovare rapidamente qualsiasi file/directory sul tuo computer.<br>Esegue la scansione periodica delle unità e genera file di dati (mappe) in modo da poter trovare rapidamente i tuoi file.<br>I file creati dopo l'ultima scansione non potranno essere trovati poiché non saranno nei file di dati, quindi si consiglia di scegliere un periodo di scansione automatica più frequente.'
```

Comment:
text_allow_global_search_message: 'La ricerca globale ti consente di trovare rapidamente qualsiasi file/directory sul tuo computer.<br>Esegue la scansione periodica delle unità e genera file di dati (mappe) in modo da poter trovare rapidamente i tuoi file.<br>I file creati dopo l\'ultima scansione non potranno essere trovati poiché non saranno nei file di dati, quindi si consiglia di scegliere un periodo di scansione automatica più frequente.'